.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+name: Build and Test
+
+
+on:
+  push:
+    branches:
+      - 'test/*'
+  pull_request:
+    branches:
+      - 'test/*'
```

Comment:
Let's drop the pull_request trigger, only run on push. Also, any reason to limit this to certain branches at this point?

Reply:
I think my preference would be to run on push to all branches except main https://stackoverflow.com/a/62918562/589391